src/protocolgame.cpp

```diff
@@ -988,6 +988,23 @@ void ProtocolGame::parseRotateItem(NetworkMessage& msg)
 	addGameTaskTimed(DISPATCHER_TASK_EXPIRATION, &Game::playerRotateItem, player->getID(), pos, stackpos, spriteId);
 }
 
+void ProtocolGame::parseRuleViolationReport(NetworkMessage &msg)
+{
+	uint8_t reportType = msg.getByte();
+	uint8_t reportReason = msg.getByte();
+	const std::string targetName = msg.getString();
+	const std::string comment = msg.getString();
+	std::string translation;
```

Comment:
@wgrriffel maybe you could enlight me, where did you get that information my friend?